contacts-jquerymobile/README.md

```diff
@@ -3,16 +3,15 @@ contacts-mobile-basic: CRUD Example Using HTML5, jQuery Mobile and JAX-RS
 Author: Joshua Wilson  
 Level: Beginner  
 Technologies: jQuery Mobile, jQuery, JavaScript, HTML5, REST  
-Summary: The `contacts-mobile-basic` quickstart demonstrates a Java EE 6 mobile database application using HTML5, jQuery Mobile, JAX-RS, JPA 2.0, and REST.  
+Summary: The `contacts-mobile-basic` quickstart demonstrates a Java EE 7 mobile database application using HTML5, jQuery Mobile, JAX-RS, JPA 2.0, and REST.  
```

Comment:
Java EE 7 uses JPA 2.1 - Maybe if we remove the versions, this kind of issue should be gone.